docs/configure/policy/rules.md

```diff
@@ -121,6 +121,45 @@ if (ActionItem.Namespace === "api") {
 }
 ```
 
+### PagerDuty Incidents
+If you have attached a PagerDuty installation to your organization, you can use the
+`createPagerDutyIncident` function to create incidents. The function takes two arguments:
+
+* from - the email address of a valid PagerDuty user on the account associated with the auth token
+* incident - an object that expects the following properties:
+  * title - a summary of the incident
+  * serviceID - the id of the service that the incident belongs to
```

Comment:
the ID of the service that the incident belongs to. 

Do we think it's better to explain where to get this ID from? Or will people familiar with pager duty know this?

Reply:
Idk, unfortunately there is no direct link to the service list page because it depends on the logged account.

- `https://vvezani.pagerduty.com/service-directory` - ✅ 
- `https://pagerduty.com/service-directory` - ⛔ 

Maybe point it out that it should be extracted from `https://<subdomain>.pagerduty.com/service-directory/PIIWGG1` → **PIIWGG1**